### ISSUE TITLE
[ENH]: Anomaly - need to include seasonality  

### ISSUE THREAD

**Issue by BalasubramaniamDakshinamoorthi** [pycaret/pycaret#2795]
### Describe the feature you want to add to this project

I need Anomaly Detection needs to take into account of seasonality , currently there is no option - included for seasonality.

### Describe your proposed solution

Needs to include  categorical features for seasonality [flags] such quarterly , half yearly , weekly , monthly flags to indicate seasonal patterns and pass them to model for training. Model will then learn seasonality patterns when identifying anomalies.

**Comment by ngupta23** [comment]
@BalasubramaniamDakshinamoorthi  Are you referring to the Anomaly Detection module or the time series module?

**Comment by BalasubramaniamDakshinamoorthi** [comment]
Anomaly Detection

**Comment by BalasubramaniamDakshinamoorthi** [comment]
@ngupta23 Anomaly Detection

**Comment by BalasubramaniamDakshinamoorthi** [comment]
@ngupta23 I need to predict Anomaly score but my data is seasonality based time series

**Comment by ngupta23** [comment]
Do you have only 1 time series and want to detect anomalies there?

**Comment by BalasubramaniamDakshinamoorthi** [comment]
yes - meaning 1 -one variable - measured during seasonal time period

**Comment by BalasubramaniamDakshinamoorthi** [comment]
time series

**Comment by ngupta23** [comment]
We do not have a module for time series anomaly detection at this time. You can consider using sktime to build this manually for now. For example, 

https://github.com/alan-turing-institute/sktime/pull/708
